src/v/transform/tests/test_fixture.cc

```diff
@@ -165,12 +172,22 @@ fake_offset_tracker::load_committed_offsets() {
     co_return _committed;
 }
 
+class commit_wait_timed_out : public ss::condition_variable_timed_out {
+    const char* what() const noexcept override {
+        return "waiting for committed offset timed out";
+    }
+};
+
 ss::future<> fake_offset_tracker::wait_for_committed_offset(
   model::output_topic_index index, kafka::offset o) {
-    return _cond_var.wait(1s, [this, index, o] {
-        auto it = _committed.find(index);
-        return it != _committed.end() && it->second >= o;
-    });
+    return _cond_var
+      .wait(
+        1s,
+        [this, index, o] {
+            auto it = _committed.find(index);
+            return it != _committed.end() && it->second >= o;
+        })
```

Comment:
```
clang-format
  really
  went for 
    it
    here!
```
I rather like it

Reply:
seriously. Not how I would have formatted it, but meh